### PR TITLE
Add space between icons and text.

### PR DESCRIPTION
**Pull request by jagtalon**
screen shot 2016-03-09 at 8 04 31 am

### CODE REVIEW

**Comment by chrismorast** [comment]
It should be .5em (margin-right: .5em; on the icon).

File: src/scss/ia/_live.scss [hunk 1]
```diff
@@ -356,6 +356,10 @@
     .ia-single--info ul li .date a {
         display: inline;
     }
+
+    .ia-single--info i {
+	margin-right: .5em;
```
Comment (MariagraziaAlastra) [inline src/scss/ia/_live.scss]: @chrismorast Isn't this the right value for margin-right?

**Comment by MariagraziaAlastra** [comment]
This is how it looks now: 
screenshot-maria duckduckgo com 5001 2016-03-15 16-36-22

**Comment by jagtalon** [comment]
@MariagraziaAlastra should be ready!

**Comment by jagtalon** [comment]
screen shot 2016-03-22 at 2 46 24 pm

**Comment by MariagraziaAlastra** [comment]
LGTM, thanks @jagtalon! :+1:

**Comment by jagtalon** [comment]
Woot

**Pull request merged**